asv_bench/benchmarks/pandas/groupby.py

```diff
@@ -0,0 +1,994 @@
+from functools import partial
```

Comment:
Size of this file scares me - we definitely don't want to benchmark everything in this file. Let's pick only useful cases

Reply:
trimmed a bunch of these. pls take a look